frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -93,58 +93,45 @@ export class VirtualMachine extends KubevirtDetailView {
     return vmView.rdpPort.getText();
   }
 
-  async create({
-    name,
-    namespace,
-    description,
-    template,
-    provisionSource,
-    operatingSystem,
-    flavor,
-    workloadProfile,
-    startOnCreation,
-    cloudInit,
-    storageResources,
-    networkResources,
-  }: VMConfig) {
+  async create(config: VMConfig) {
```

Comment:
what is the reason for this? does it override some other global constants? The old way is a bit easier on the eyes IMO.

Reply:
Indeed... reverted